src/routes/+layout.svelte

```diff
@@ -86,13 +86,28 @@
 								})}>Teams</a
 							>
 						</li>
+						<li>
+							<div class="">
+								<a href="/qa" class="font-bold tracking-wide uppercase"> faq </a>
```

Comment:
switch the order of the these two

Reply:
![image](https://user-images.githubusercontent.com/43375532/214309198-d13ffb1b-8aac-4abd-8a63-5d8758d63be2.png)
also, they are in the wrong place, check screenshot